java/integTest/src/test/java/glide/TestConfiguration.java

```diff
@@ -1,18 +1,58 @@
-/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+/**
+ * Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+ */
 package glide;
 
 import com.vdurmont.semver4j.Semver;
+import glide.api.GlideClient;
+import glide.api.models.commands.InfoOptions;
+
 import java.util.Arrays;
 
+import static glide.TestUtilities.commonClientConfig;
+
 public final class TestConfiguration {
     // All servers are hosted on localhost
     public static final int[] STANDALONE_PORTS = getPortsFromProperty("test.server.standalone.ports");
     public static final int[] CLUSTER_PORTS = getPortsFromProperty("test.server.cluster.ports");
-    public static final Semver SERVER_VERSION = new Semver(System.getProperty("test.server.version"));
+    private static final GlideClient standaloneClient;
+    public static final Semver SERVER_VERSION;
+
+    static {
```

Comment:
Please add a function to `TestUtilities` which runs `info` and parses the response. The function has one argument - client.
Then static block will call that function. Add `@SneakyThrows` to the function - we don't care about exceptions.

This could be used in some specific tests where we could have servers of different versions.

Reply:
Updated.
Moved function to TestUtilities.